src/main/java/net/snowflake/client/log/StdOutConsoleHandler.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2012-2019 Snowflake Computing Inc. All rights reserved.
+ */
+package net.snowflake.client.log;
+
+import java.util.logging.LogRecord;
+import java.util.logging.SimpleFormatter;
+import java.util.logging.StreamHandler;
+
+class StdOutConsoleHandler extends StreamHandler {
+  public StdOutConsoleHandler() {
+    // configure with specific defaults for ConsoleHandler
+    super(System.out, new SimpleFormatter());
+  }
+
+  @Override
+  public void publish(LogRecord record) {
+    super.publish(record);
+    flush();
```

Comment:
Do we really want to flush that often? What is the purpose? What is the impact on perf here?

Reply:
It's implemented in the same way in [ConsoleHandler](https://github.com/AdoptOpenJDK/openjdk-jdk11/blob/master/src/java.logging/share/classes/java/util/logging/ConsoleHandler.java)